priv/riak_core.schema

```diff
@@ -200,3 +201,56 @@
     {datatype, {duration, ms}},
     hidden
 ]}.
+
```

Comment:
The schema elements should be tested in `riak_core_schema_tests.erl.` You may find some useful examples in the riak_kv multibackend schema tests.